### ISSUE TITLE
Design Chrome Web Store promotional images

### ISSUE THREAD

**Issue by ggrossetie** [asciidoctor/brand#12]
Layout constraints:

Small – 440 x 280 :
Large – 920 x 680 : (optional)
Scrolling banner – 1 400 x 560 : (optional)

**current**
![current](https://user-images.githubusercontent.com/333276/47136876-4dbdd480-d2b5-11e8-91d1-cf95ec80b987.png)

**examples**
![ex](https://user-images.githubusercontent.com/333276/47136828-2a932500-d2b5-11e8-85ae-eab3643c7de3.png)

**Comment by ggrossetie** [comment]
@mojavelinux Maybe I can add the Twitter banner as a promotional image ? 

https://twitter.com/asciidoctor
https://pbs.twimg.com/profile_banners/1040792407/1539565274/1500x500

**Comment by mojavelinux** [comment]
Yes, feel free to use that for now. I'll submit a PR with the draft SVG.

**Comment by mojavelinux** [comment]
see https://github.com/asciidoctor/brand/pull/14
